res/menu/file_actions_menu.xml

```diff
@@ -17,72 +17,82 @@
   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <item
         android:id="@+id/action_share_file"
         android:title="@string/action_share"
         android:icon="@drawable/ic_action_share"
-        android:orderInCategory="1" />
-
-    <item
-        android:id="@+id/action_open_file_with"
-        android:title="@string/actionbar_open_with"
-        android:icon="@drawable/ic_open_in_app"
+        app:showAsAction="ifRoom"
         android:orderInCategory="1" />
     <item
         android:id="@+id/action_download_file"
         android:title="@string/filedetails_download"
         android:icon="@drawable/ic_action_download"
+        app:showAsAction="ifRoom"
         android:orderInCategory="1" />
     <item
         android:id="@+id/action_sync_file"
         android:title="@string/filedetails_sync_file"
         android:icon="@drawable/ic_action_refresh_white"
+        app:showAsAction="ifRoom"
         android:orderInCategory="1" />
     <item
         android:id="@+id/action_cancel_sync"
         android:title="@string/common_cancel_sync"
         android:icon="@drawable/ic_action_cancel_white"
+        app:showAsAction="ifRoom"
         android:orderInCategory="1" />
     <item
-        android:id="@+id/action_rename_file"
-        android:title="@string/common_rename"
-        android:icon="@drawable/ic_pencil"
+        android:id="@+id/action_remove_file"
+        android:title="@string/common_remove"
+        android:icon="@drawable/ic_action_delete_white"
+        app:showAsAction="ifRoom"
+        android:orderInCategory="1" />
+    <item
+        android:id="@+id/action_copy"
+        android:title="@android:string/copy"
+        android:icon="@drawable/ic_action_copy"
+        app:showAsAction="ifRoom"
         android:orderInCategory="1" />
     <item
         android:id="@+id/action_move"
         android:title="@string/actionbar_move"
         android:icon="@drawable/ic_action_move"
+        app:showAsAction="ifRoom"
         android:orderInCategory="1" />
     <item
-        android:id="@+id/action_copy"
-        android:title="@android:string/copy"
-        android:icon="@drawable/ic_action_copy"
+        android:id="@+id/action_open_file_with"
+        android:title="@string/actionbar_open_with"
+        android:icon="@drawable/ic_open_in_app"
+        app:showAsAction="never"
         android:orderInCategory="1" />
     <item
-        android:id="@+id/action_remove_file"
-        android:title="@string/common_remove"
-        android:icon="@drawable/ic_action_delete_white"
+        android:id="@+id/action_rename_file"
```

Comment:
I changed the order of the elements in the menu, moving first those I think could be used more often. But remove is still up there :)